### PR TITLE
Update callRecords changelog with new sessions pageSize

### PR DESCRIPTION
**Pull request by mcm223**
The callRecords API now has a maximum page size of 60 for returned sessions in both beta and v1.0.

### CODE REVIEW

**Comment by Danielabom** [comment]
Contacted author to get info.

File: changelog/Microsoft.IC3.DataPlatform.json [hunk 1]
```diff
@@ -1,5 +1,41 @@
 {
     "changelog": [
+        {
+            "ChangeList": [
+                {
+                    "Id": "588cac11-3d88-4e1d-a284-af3b1ceb6cbf",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "sessions",
+                    "ChangeType": "Change",
+                    "Description": "Change maximum page size to 60 sessions for a [callRecords query](https://learn.microsoft.com/en-us/graph/api/resources/callrecords-api-overview?view=graph-rest-1.0).",
```
Comment (Danielabom) [inline changelog/Microsoft.IC3.DataPlatform.json:10]:
```suggestion
                    "Description": "Change the maximum page size to 60 sessions in a query for the [callRecords](https://learn.microsoft.com/en-us/graph/api/resources/callrecords-api-overview?view=graph-rest-1.0) resource.",
```

File: changelog/Microsoft.IC3.DataPlatform.json [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+            "SubArea": "Calls and online meetings"
+        },
+        {
+            "ChangeList": [
+                {
+                    "Id": "62e788b4-e9ae-42ab-8318-a7732199d52c",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "sessions",
+                    "ChangeType": "Change",
+                    "Description": "Change maximum page size to 60 sessions for a [callRecords query](https://learn.microsoft.com/en-us/graph/api/resources/callrecords-api-overview?view=graph-rest-beta).",
```
Comment (Danielabom) [inline changelog/Microsoft.IC3.DataPlatform.json:28]:
```suggestion
                    "Description": "Changed the maximum page size to 60 sessions in a query for the [callRecords](https://learn.microsoft.com/en-us/graph/api/resources/callrecords-api-overview?view=graph-rest-beta) resource.",
```

**Review by Danielabom — CHANGES_REQUESTED** [review]
Thank you for working on this PR.  I added some comments, please review them. Just one more thing: I assume that the previous size was expected or the default behavior. Wouldn't updating the resource's topic with a note be important?

**Comment by mcm223** [comment]
Thank you @Danielabom! I've committed your suggested language changes.

> I assume that the previous size was expected or the default behavior. Wouldn't updating the resource's topic with a note be important?

I reviewed our other API docs and we don't currently specify maximum page sizes for any of them. We do make a more general comment that truncated results will have a ```odata.nextLink``` value. I have a rather large batch of doc updates planned so I can include max page sizes for all our endpoints in a future PR, as I agree I think it would be good to have them all explicitly listed, but I don't have them all handy at the moment and I'd prefer not to have them inconsistently documented.

**Review by Danielabom — APPROVED** [review]
Thank you for working on this PR and thank you for addressing my comments. PR approved. Thanks.

File: changelog/Microsoft.IC3.DataPlatform.json [hunk 3]
```diff
@@ -1,5 +1,41 @@
 {
     "changelog": [
+        {
+            "ChangeList": [
+                {
+                    "Id": "588cac11-3d88-4e1d-a284-af3b1ceb6cbf",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "sessions",
+                    "ChangeType": "Change",
+                    "Description": "Changed the maximum page size to 60 sessions in a query for the [callRecords](https://learn.microsoft.com/en-us/graph/api/resources/callrecords-api-overview?view=graph-rest-1.0) resource.",
```
Comment (Lauragra) [inline changelog/Microsoft.IC3.DataPlatform.json:10]: This is a link to the API overview topic, not the callRecords resource topic. I don't see anywhere in either the callRecords or the session resource topics where we mention a maximum page size. Please add updates to the associated documentation to this PR before we merge, and update the links to target the appropriate updated docs. Thanks.

Comment (mcm223) [reply changelog/Microsoft.IC3.DataPlatform.json]: I've updated the links and added the page size where we discuss pagination in getting the call record.

File: changelog/Microsoft.IC3.DataPlatform.json [hunk 4]
```diff
@@ -1,5 +1,41 @@
...
     "changelog": [
+        {
+            "ChangeList": [
+                {
+                    "Id": "588cac11-3d88-4e1d-a284-af3b1ceb6cbf",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "sessions",
+                    "ChangeType": "Change",
+                    "Description": "Changed the maximum page size to 60 sessions in a query for the [callRecords](https://learn.microsoft.com/en-us/graph/api/resources/callrecords-callrecord?view=graph-rest-1.0) resource.",
+                    "Target": "callRecord"
```
Comment (Danielabom) [inline changelog/Microsoft.IC3.DataPlatform.json:11]:
```suggestion
                    "Target": "callRecords"
```

File: changelog/Microsoft.IC3.DataPlatform.json [hunk 5]
```diff
@@ -1,5 +1,41 @@
...
+        },
+        {
+            "ChangeList": [
+                {
+                    "Id": "62e788b4-e9ae-42ab-8318-a7732199d52c",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "sessions",
+                    "ChangeType": "Change",
+                    "Description": "Changed the maximum page size to 60 sessions in a query for the [callRecords](https://learn.microsoft.com/en-us/graph/api/resources/callrecords-callrecord?view=graph-rest-beta) resource.",
+                    "Target": "callRecord"
```
Comment (Danielabom) [inline changelog/Microsoft.IC3.DataPlatform.json:29]:
```suggestion
                    "Target": "callRecords"
```

**Review by Danielabom — APPROVED** [review]
@Lauragra, it seems that all the changes you requested are in. Thanks!

**Comment by Danielabom** [comment]
@mcm223, thank you for addressing our comments. Your PR is almost ready; however, the build keeps failing. Please look into it.

**Pull request merged**